default.nix

```diff
@@ -50,6 +51,17 @@ in
           sha256 = "13lim8vv78m9lhn7qfjswg7ax825gn0v75gcb80hckxawgk8zxc1";
         };
 
+        systemd =
+          if systemdFeatures then
+            callHackageDirect {
+              pkg = "systemd";
+              # we'd like to use a newer one but it requires network >= 3
+              ver = "1.2.0";
+              sha256 = "1mwrrki3zsc4ncr7psjv9iqkzh7f25c2ch4lf2784fh6q46i997j";
+            }
+          else null;
```

Comment:
`null` is okay?

Reply:
The osx nix build wouldn't have passed otherwise